engine/src/externalv1.cpp

```diff
@@ -967,44 +967,17 @@ Exec_stat MCExternalV1::Handle(MCObject *p_context, Handler_type p_type, uint32_
 	
 	// Now iterate through the parameters, fetching the parameter values as we go.
 	for(uint32_t i = 0; p_parameters != nil && t_stat == ES_NORMAL; i++, p_parameters = p_parameters -> getnext())
-	{
-		MCVariable* t_var;
-		t_var = p_parameters -> eval_argument_var();
-		if (t_var != nil)
-		{
-			// We are a variable parameter so use the value directly (i.e. pass by
-			// reference).
-			
-			// MW-2014-01-22: [[ CompatV1 ]] Create a reference to the MCVariable.
-			t_parameter_vars[i] = new (nothrow) MCReferenceExternalVariable(t_var);
-		}
-		else
-		{
-			// AL-2014-08-28: [[ ArrayElementRefParams ]] Evaluate container if necessary
-			MCAutoValueRef t_value;
-			MCContainer *t_container;
-			t_container = p_parameters -> eval_argument_container();
-			
-			if (t_container != nil)
-			{
-				MCNameRef *t_path;
-				uindex_t t_length;
-				t_container -> getpath(t_path, t_length);
-				
-				MCExecContext ctxt(p_context, nil, nil);
-				
-				if (t_length == 0)
-					t_parameter_vars[i] = new (nothrow) MCReferenceExternalVariable(t_container -> getvar());
-				else
-					t_container -> eval(ctxt, &t_value);
-			}
-			else
-				t_value = p_parameters -> getvalueref_argument();
-			
-			// MW-2014-01-22: [[ CompatV1 ]] Create a temporary value var.
-			if (*t_value != nil)
-				t_parameter_vars[i] = new (nothrow) MCTransientExternalVariable(*t_value);
-		}
+    {
+        MCContainer *t_container;
+        t_container = p_parameters -> eval_argument_container();
```

Comment:
I'm not sure it is possible as the MCParameter owns the MCContainer, and the MCReferenceExternalVariable will share it temporarily.